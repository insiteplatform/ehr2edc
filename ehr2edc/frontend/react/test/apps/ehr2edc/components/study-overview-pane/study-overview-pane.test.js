import React from 'react';
import renderer from 'react-test-renderer';
import StudyOverviewPane from "../../../../../src/apps/ehr2edc/components/study-overview-pane/study-overview-pane";

describe("Rendering the component", () => {
    test("Loading the company with a study", () => {
        const study = {
            id: "study-1",
            name: "Test study nr. 1",
            description: "Description for 'Test study nr. 1'"
        };
        const app = renderer.create(<StudyOverviewPane study={study} />);
        const tree = app.toJSON();
        expect(tree).toMatchSnapshot();
    });
    test("Loading the company without a study", () => {
        const app = renderer.create(<StudyOverviewPane />);
        const tree = app.toJSON();
        expect(tree).toMatchSnapshot();
    });
});