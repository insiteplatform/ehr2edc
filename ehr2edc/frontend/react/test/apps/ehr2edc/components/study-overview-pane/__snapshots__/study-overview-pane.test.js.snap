// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering the component Loading the company with a study 1`] = `
<div
  className="grid-content margin"
>
  <h3
    className="subheader"
  >
    study-1
     — 
    Test study nr. 1
  </h3>
  <p
    className="margin-tb"
  >
    Description for 'Test study nr. 1'
  </p>
</div>
`;

exports[`Rendering the component Loading the company without a study 1`] = `
<div
  className="grid-content margin"
>
  <h3
    className="subheader"
  >
     — 
  </h3>
  <p
    className="margin-tb"
  />
</div>
`;
